index.js

```diff
@@ -0,0 +1,49 @@
+import Book from './modules/Book.js'; /* eslint-disable-line */
```

Comment:
- [ ] It is not good practice not to disable linters, linting is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. If part of your code breaks the standard rules, this can pose a problem, please avoid disabling linters for all your files